src/benchmark/RemotePingPong/Program.cs

```diff
@@ -57,6 +61,11 @@ public static Config CreateActorSystemConfig(string actorSystemName, string ipOr
                     port = 0
                     hostname = ""localhost""
                 }
+
+                default-remote-dispatcher = {
```

Comment:
RemotePingPong numbers running two different `Channel<T>` dispatchers were about half of what they are in `dev`